docs/source/examples/de_sscan.ipynb

```diff
@@ -0,0 +1,1505 @@
+    "        scan1.positioners.p1.setpoint_pv, m1.user_setpoint.pvname,\n",
+    "    )\n",
+    "    # triggers (push scaler count button at each point)\n",
+    "    yield from bps.mv(\n",
+    "        scan1.triggers.t1.trigger_pv, scaler1.count.pvname,\n",
+    "    )\n",
+    "    # detectors\n",
+    "    yield from bps.mv(\n",
+    "        scan1.detectors.d01.input_pv, I0.pvname,\n",
+    "        scan1.detectors.d02.input_pv, diode.pvname,\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Setup the scan."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "RE(setup_scan1(-1.1, 1.1, 11, 0.2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the scan.  We should not have to reprogram this plan."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "RE(run_sscan())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Get the scan data.  Same function as before.  It's not a bluesky plan, so `RE()`\n",
+    "is not needed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'m1': array([-1.1 , -0.88, -0.66, -0.44, -0.22,  0.  ,  0.22,  0.44,  0.66,\n",
+       "         0.88,  1.1 ]),\n",
+       " 'I0': array([2., 1., 1., 2., 1., 3., 2., 1., 1., 1., 0.], dtype=float32),\n",
+       " 'diode': array([1., 2., 1., 2., 2., 4., 4., 2., 0., 1., 1.], dtype=float32)}"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "get_sscan_data()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Post data to bluesky as a run\n",
+    "\n",
+    "So far, we have just run `scan1` and shown the data collected.  We'll need to do\n",
+    "a bit more work to get this data into bluesky as a run.  While we might just use\n",
+    "`bps.read(scan1)` to get the data, the data would be named according to the\n",
+    "`scan1` structure.  We want more meaningful names so we re-assign the names of\n",
+    "the original objects (motor and scaler channel names).  To post *bluesky data*,\n",
+    "it must come from an ophyd `Device` (subclass).  We'll create a custom one just\n",
+    "for our setup of `scan1`, then `bps.mv()` each of the arrays.  Since we do not\n",
```

Comment:
I do not understand this "then `bps.mv()` each of the arrays", is it missing a word?